test_apps/contracts_app/blockchain.json

```diff
@@ -25,7 +25,18 @@
   "testnet": {
     "networkType": "testnet",
     "rpcHost": "localhost",
-    "rpcPort": 8545
+    "rpcPort": 8545,
+    "account": {
+      "password": "config/development/password"
```

Comment:
Should use separate passwords, no?

Reply:
you could. I don't think it's necessary for this testing account though. Do you think it is just for displaying good behavior? That's the only pro I can think out of this.

Reply:
Yeah I guess that would be the only pro. Just in case someone were to get inspiration from those config files

Reply:
Alright. I'll make the change.